pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
-
+    mkdir -p "$out/nix-support"
     for drv in $unwrapped; do
-        if [ -a "$drv/nix-support/propagated-user-env-packages" ]; then
-            mkdir -p "$out/nix-support"
-            cat "$drv/nix-support/propagated-user-env-packages" >> "$out/nix-support/propagated-user-env-packages"
-        fi
+        echo "$drv" >> "$out/nix-support/propagated-user-env-packages"
```

Comment:
I thought so too and initially wanted to set higher priority for the wrapper to remedy this. However, experiment showed that somehow that's not the case:

```
$ nix-build -E 'with import ./. {}; buildEnv { name = "foo"; paths = [ kde5.gwenview ]; ignoreCollisions = false; checkCollisionContents = true; }'
```

completed without any collision warnings. I'm still wary of this however (that's part of why I'm unsure if this is a good idea). It solves problem of linking unwrapped `/share` however.